mob.go

```diff
@@ -645,6 +680,21 @@ func getCachedChanges() string {
 	return strings.TrimSpace(silentgit("diff", "--cached", "--stat"))
 }
 
+func makeWipCommit() {
+	git("add", "--all")
+	wipCommitMsg := configuration.WipCommitMessage
+	stagedCoauthors := gitStagedCoauthors()
```

Comment:
can you extract a method that gets the configured wip commit message and adds the coauthors? Mabye name it addCoAuthors?